books/en_US/src/c01-01-setup.md

```diff
@@ -68,7 +68,7 @@ more dependencies. If this step fails and you see errors related to `alsa` and `
 Now let's actually use ggez in the main file and set up our window. This is just the simplest example of a ggez program that will give us a window with nothing else. Copy and paste this into the `main.rs` file and run again.
 
 ```rust
-{{#include ../../../code/rust-sokoban-c01-01/src/main.rs}}
+{{#include ../code/rust-sokoban-c01-01/src/main.rs:all}}
```

Comment:
I think you might need to find/replace `../code -> ../../../code`, this is because of the previous PR which moved files around in subdirectories 😄